File: 2018/J1.py
lst = []
for i in range (4):
    num = int(input())
    lst.append(num)
if lst[0] == 8 or lst[0] == 9 and lst[3] ==8 or lst[3] ==9 and lst[1] == lst[2]:
    print ("ignore")
else:
    print ("answer")

File: 2018/Java/J1.java
import java.util.*;
public class J1 {
	public static void main(String args[]) {
        Scanner input = new Scanner(System.in);
        int n1 = input.nextInt();
        int n2 = input.nextInt();
        int n3 = input.nextInt();
        int n4 = input.nextInt();
        if ((n1 == 8 || n1 == 9) && (n4 == 8 || n4 == 9) && (n2 == n3)){
            System.out.println("ignore");
        }
        else{
            System.out.println("answer");
        }
    }
}
